src/core/Solver_intf.ml

```diff
@@ -13,52 +13,47 @@ Copyright 2016 Simon Cruanes
 
 type 'a printer = Format.formatter -> 'a -> unit
 
+(** The type of values returned when the solver reaches a SAT state. *)
 type ('term, 'form, 'value) sat_state = {
-  eval: 'form -> bool;
-  (** Returns the valuation of a formula in the current state
+  eval : 'form -> bool;
+      (** Returns the valuation of a formula in the current state
```

Comment:
Why is there indentation in these doc comments ? (I mean what would be ocamlformat's rationale for that ?)